CHANGES.rst

```diff
@@ -1,3 +1,28 @@
+2.3.0 (2024-12-22)
+==================
+
+Features
+--------
+
+- Added ``HTTPResponse.shutdown()`` to stop any ongoing or future reads for a specific response. It calls ``shutdown(SHUT_RD)`` on the underlying socket. This feature was `sponsored by LaunchDarkly <https://opencollective.com/urllib3/contributions/815307>`__. (`#2868 <https://github.com/urllib3/urllib3/issues/2868>`__)
+- Added support for JavaScript Promise Integration on Emscripten. This enables more efficient WebAssembly 
+  requests and streaming, and makes it possible to use in Node.js if you launch it as  ``node --experimental-wasm-stack-switching``. (`#3400 <https://github.com/urllib3/urllib3/issues/3400>`__)
+
+
+Bugfixes
+--------
+
+- Added the ``proxy_is_tunneling`` property to ``HTTPConnection`` and ``HTTPSConnection``. (`#3285 <https://github.com/urllib3/urllib3/issues/3285>`__)
+- Added pickling support to ``NewConnectionError`` and ``NameResolutionError``. (`#3480 <https://github.com/urllib3/urllib3/issues/3480>`__)
```

Comment:
This two items look like features rather than bugfixes, sorry I missed this during reviews

Reply:
Thank you, done in [`bea61c7` (#3536)](https://github.com/urllib3/urllib3/pull/3536/commits/bea61c7763066211326b50691fefc3d09d561817).

Reply:
Thanks for moving them!